File: official/docs/go/users/create_child_user.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
    client := easypost.New(apiKey)

	userName := "Child Account Name"
    user, err := client.CreateUser(
        &easypost.UserOptions{
            Name: &userName,
        },
    )

	fmt.Println(user)
}

File: official/docs/csharp/current/users/create-child-user.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;
using EasyPost.Models.API;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

            var client = EasyPost.Client(apiKey);

            User user = await client.User.Create(new Dictionary<string, object>()
            {
                { "name", "Child Account Name" }
            });

            Console.WriteLine(JsonConvert.SerializeObject(user, Formatting.Indented));
        }
    }
}
